meta-fobnail-distro/recipes-fobnail/qcobr/qcbor_git.bb

```diff
@@ -0,0 +1,36 @@
+LICENSE = "CLOSED"
```

Comment:
@kotylamichal please add description and homepage, license info here https://github.com/laurencelundblade/QCBOR#copyright-and-license

Reply:
done